splash.py

```diff
@@ -6,6 +6,11 @@
 def splash(i, o):
     image = PIL.Image.open("splash.png").convert('L')
     image = invert(image)
-    image = image.convert(o.device.mode)
```

Comment:
For now, `if hasattr(o, "device")` seems like a good temporary workaround - until the #3 changes will be merged, then there'll be a more abstract API. As for now, PIL image direct output to device is only used by the splash screen itself, so until UI elements start generating PIL objects themselves (they eventually will, to have a prettier and more customizable UI), we don't need this feature in emulator that much and can just workaround it (but it will be necessary right once #3 is done).